adcc/smoketests/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
```

Comment:
Why is this file needed at all?

Reply:
I wanted to invoke with `pytest --pyargs adcc.smoketests`, but can as well go for `pytest --pyargs adcc`. Probably the latter version is better (= shorter)...